apps/amo/tests/test_views.py

```diff
@@ -390,3 +390,13 @@ def test_contribute_json(self):
         res = self.client.get('/contribute.json')
         eq_(res.status_code, 200)
         eq_(res._headers['content-type'], ('Content-Type', 'application/json'))
+
+
+class TestRobots(amo.tests.TestCase):
+
+    def test_disable_collections(self):
+        """Make sure /en-US/firefox/collections/ gets disabled"""
+        url = reverse('collections.list')
+        response = self.client.get('/robots.txt')
+        eq_(response.status_code, 200)
```

Comment:
@magopian - the rest of the file has `eq_()` so should this use that so there's consistency in the file? Or is the plan to start moving to pytest `assert` for all new tests/code?